components/bl-barcode-component/src/index.js

```diff
@@ -0,0 +1,79 @@
+import { useCallback, useEffect, useMemo, useRef } from 'react';
+
+import './lib/barcode.min';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const FontOptions = {
+  UNSET      : 'unset',
+  BOLD       : 'bold',
+  ITALIC     : 'italic',
+  BOLD_ITALIC: 'boldItalic',
+};
+
+const FontOptionsMap = {
+  [FontOptions.UNSET]      : '',
```

Comment:
I'm not sure, but we have an issue with empty string at select trait's in default components. To check if it not reproduced try to: Reload Page change value from 'bold' to '' in settings and check in preview if it changed and visa verse(reload, change from '' to 'bold' and check again)

Reply:
Everything works fine.
We don't use an empty string at ui builder select, we use an 'unset' value for it. And when it's selected, we pass an empty string in the library's font parameters.